discord/guild.go

```diff
@@ -29,7 +33,91 @@ type GuildState struct {
 	Tracking map[string]Tracking
 
 	//UNUSED right now
-	TextChannelId string
+	TextChannelID string
+
+	// For voice channel movement
+	MoveDeadPlayers bool
+}
+
+// TrackedMemberAction struct
+type TrackedMemberAction struct {
+	mute          bool
+	move          bool
+	message       string
+	targetChannel Tracking
+}
+
+func (guild *GuildState) handleTrackedMembers(dg *discordgo.Session, inGame bool, inDiscussion bool) {
+	// first we need to determine if we care about moving people at all
+	if guild.MoveDeadPlayers {
+		guild.moveAndMuteAllTrackedMembers(dg, inGame, inDiscussion)
+		return
+	}
+
+	guild.muteAllTrackedMembers(dg, inGame, inDiscussion)
+}
+
+func (guild *GuildState) moveAndMuteAllTrackedMembers(dg *discordgo.Session, inGame bool, inDiscussion bool) {
+	guild.voiceStatusCacheLock.RLock()
+
+	deadUserChannel, deadUserChannelError := guild.findVoiceChannel(true)
+	if deadUserChannelError != nil {
```

Comment:
I had considered making these error out earlier, but I think it is okay to do here. Probably worth adding some message back to the user as well.